.github/workflows/build_site.yml

```diff
@@ -11,12 +11,18 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/cache@v2 # FIXME: add pip + latex + gems
+      - uses: actions/cache@v2 # FIXME: add apt(latex)
```

Comment:
does this still need to be done?

Reply:
The cache? It makes the building stage to use what we have downloaded before instead of keeping downloading once and again the same thing (unless we require a newer version).

Reply:
oh sorry should have been more specific there, the `# FIXME: add apt(latex)`

Reply:
Yes, but it could be done later when we know how...